pyiron_atomistics/atomistics/job/atomistic.py

```diff
@@ -891,3 +892,15 @@ def __dir__(self):
             return hdf5_path.list_nodes()
         else:
             return []
+
+    @property
+    def mean_squared_displacement(self):
+        """
+        The mean-squared displacement computed directly within Lammps.
+
+        WARNINGS:
+            1. This works only if the job is a `Lammps` job.
+            2. Returns an output ONLY when the `msd` flag in set to True in `calc_md` for a Lammps job.
+            3. Returns None if used with `calc_static`, `calc_minimize` and `calc_vcsgc` for a Lammps job.
```

Comment:
I share @samwaseda's concerns about cluttering the Lammps codebase when this feature is so specific. What is the exact use-case you have in mind (I think I know -- FTS?) and what is the cost savings compared to post-processing in python from unwrapped positions?

Reply:
> Right, so getting the MSD from the unwrapped positions takes roughly about 550ms, for a structure with 500 atoms, with 10k snapshots.

As I mentioned in a previous comment, the cost for smaller structures is not worth fretting over. I do agree with Sam's and your concerns over cluttering the Lammps codebase with something that can be done outside of Lammps. 

The use case I had in mind was definitely to calculate diffusion coefficients, as many Lammps users are more familiar with the `compute msd` command in the script. I also wanted to calculate approximate spring constants from the msd as mentioned in [this paper](https://doi.org/10.1016/j.commatsci.2015.10.050).